gamemodes/cinema/gamemode/modules/theater/services/sh_bitchute.lua

```diff
@@ -0,0 +1,75 @@
+local SERVICE = {}
+
+SERVICE.Name 		= "Bitchute"
+
+SERVICE.NeedsCodecs = true
+
+function SERVICE:GetKey( url )
+	match = string.match(url.path,"/.+/(.+[^/])")
+	if match != nil and string.find(url.host,"bitchute.com") then
+		return match
+	end
+	return false
+end
+
+if CLIENT then
+	function SERVICE:GetVideoInfoClientside(key, callback)
```

Comment:
You're not calling theater.GetVideoInfoClientside in the serverside part (see sv_soundcloud), so this entire function is unused